calico_containers/calico_ctl/node.py

```diff
@@ -26,7 +26,7 @@
 Options:
   --force                   Stop the node process even if it has active endpoints.
   --node-image=<DOCKER_IMAGE_NAME>    Docker image to use for Calico's per-node
-                                      container [default: calico/node:latest]
+                                      container. Default is calico/node:latest.
```

Comment:
I would move the comment below up here so that we talk about the default being X, or with libnetwork Y.


Then the comment for --libnetwork can just be run calico node with the libnetwork plugin.